supplier_app/models.py

```diff
@@ -63,6 +63,18 @@ class TaxPayerArgentina(TaxPayer):
     cuit = models.CharField(max_length=200)
     justificacion = models.CharField(max_length=200)
     forma_de_pago = models.CharField(max_length=200)
+    inscripcion_AFIP_file = models.FileField(
```

Comment:
Please use english for this model, even if we are handling `TaxPayerArgentina`

Reply:
Resolved